app/api-v1/services/recipeService.js

```diff
@@ -1 +1,15 @@
-module.exports = {}
+const { addRecipe } = require('../../db')
+
+async function createRecipe(reqBody) {
+  if (!reqBody) {
+    return null
```

Comment:
is this is a duplication of?
```js
      if (!req.body) {
        return res.status(400).json({ message: 'Invalid request' })
```

Also can we return an instance of error  `throw`

Reply:
I think I would prefer a throw yes. `null` doesn't really say anything about what the error is

Reply:
I agree, this is an error scenario for which I have no validation as yet. Is this where we need to add the lookups for the other pieces you mentioned earlier @mattdean-digicatapult 

I will try to restructure this to return an error as you suggest and then we can merge and keep going...